packages/dd-trace/src/git_metadata_tagger.js

```diff
@@ -0,0 +1,16 @@
+const { SCI_COMMIT_SHA, SCI_REPOSITORY_URL } = require('./constants')
+
+class GitMetadataTagger {
+  constructor (config) {
+    this._config = config
+  }
+
+  tagGitMetadata (spanContext) {
+    if (this._config.gitMetadataEnabled) {
+      spanContext._trace.tags[SCI_COMMIT_SHA] = this._config.commitSHA
```

Comment:
this trace tags are only added to the local root in `format.js` via `extractChunkTags`